specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/dataVersion.json

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
Just curious why this isn't included in the previous spec - seems like data versions are a direct child of data containers

Reply:
Please add a little more detail, not following the question

Reply:
Just wondering why these are separate files - generally the file split would be at the tracked (top level)  resource level